exercises/practice/largest-series-product/src/test/groovy/LargestSeriesProductSpec.groovy

```diff
@@ -0,0 +1,212 @@
+import spock.lang.*
+
+class ExerciseSpec extends Specification {
+    @IgnoreRest
```

Comment:
Exercism prefers tests be enabled one at a time in a TDD fashion. Can we do @Ignore for each of the subsequent tests instead of @IgnoreRest here? That’ll match what has been done on other test suites on this track.

Reply:
Added an ability to generate `@Ignore` attributes to test generator.